### PR TITLE
Cut(eos_cli_config_gen): Remove deprecated keys enable_vrfs and octa  from management_api_gnmi data model

### PR DESCRIPTION
**Pull request by Vibhu-gslab**
## Change Summary

Remove deprecated keys enable_vrfs and octa  from management_api_gnmi data model

## Related Issue(s)

Fixes #

## Component(s) name

`arista.avd.eos_cli_config_gen`

## Proposed changes
added removed: true in schema and removed j2 code also made modifications to deprecated_vars molecule

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: ansible_collections/arista/avd/docs/porting-guides/5.x.x.md [hunk 1]
```diff
@@ -66,8 +66,8 @@ The following data model keys have been removed from `eos_cli_config_gen` in v5.
 | event_handlers.action_type | event_handlers.actions |
 | event_handlers.regex | event_handlers.trigger_on_logging.regex |
 | old key 3(flow_trackings) | new key(TODO) |
-| old key 4.1(management_api_gnmi) | new key(TODO) |
-| old key 4.2(management_api_gnmi) | new key(TODO) |
+| enable_vrfs | transport.grpc |
+| octa | provider |
```
Comment (MaheshGSLAB) [inline ansible_collections/arista/avd/docs/porting-guides/5.x.x.md]:
```suggestion
| management_api_gnmi.enable_vrfs | management_api_gnmi.transport.grpc |
| management_api_gnmi.octa | management_api_gnmi.provider |
```

File: python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/management-api-gnmi.j2 [hunk 2]
```diff
@@ -9,20 +9,6 @@
...
-{%             set octa = 'disabled' %}
-{%         endif %}
-
-| VRF with gNMI | OCTA |
-| ------------- | ---- |
-{%         for vrf in management_api_gnmi.enable_vrfs | arista.avd.natural_sort %}
-| {{ vrf.name }} | {{ octa }} |
-{%         endfor %}
-{%     endif %}
 {# new table view using the new flags #}
```
Comment (laxmikantchintakindi) [inline python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/management-api-gnmi.j2]:
```suggestion
```

**Review by laxmikantchintakindi — APPROVED** [review]

**Comment by laxmikantchintakindi** [comment]
We can remove lines 124-139 from schema file and its documentation.

**Comment by laxmikantchintakindi** [comment]
line 148 too.

**Review by MaheshGSLAB — DISMISSED** [review]
You should remove the below files
python-avd/pyavd/_eos_cli_config_gen/schema/eos_cli_config_gen.jsonschema.json
python-avd/pyavd/_eos_designs/schema/eos_designs.jsonschema.json

**Review by gmuloc — APPROVED** [review]
LGTM

**Pull request merged**